docs/actions.md

```diff
@@ -31,6 +31,7 @@ Learn how to create, invoke, and debug actions in your preferred development env
 In addition, learn about:
 
 * [Watching action output](#watching-action-output)
+* [Getting actions](#getting-actions)
```

Comment:
currently, only one action can be retrieved at a time... plural not needed here..?  i.e. `Getting an action` 

`Getting` vs `Retrieving` or `Obtaining`..?

Reply:
@mdeuser, was just following the pattern that is already use in the doc. For instance, see `Deleting actions`.

Reply:
ok. makes sense.

Reply:
oh.. maybe move `Accessing action metadata within the action body` underneath this `Getting Actions` section since it's all related to action retrieval.

Reply:
`Accessing action metadata within the action body` seems to be associated with action environment variables, which is not related to the action get command.

Reply:
ah.  my bad.  i misread the section header and was thinking of the action metadata associated with the action (i.e. memory limit, timeout, etc)..  

ihmo, it's the use of the unqualified word `metadata` that lead to my confusion.  it's used in the `Getting actions` section and in the `Accessing action metadata...` section, but each represent a different set of properties.  maybe the qualification is `action configuration metadata` vs `action runtime metadata`.